packages/fxa-settings/src/components/Settings/ConnectedServices/index.tsx

```diff
@@ -239,6 +240,18 @@ export const ConnectedServices = () => {
           </Localized>
         </div>
 
+        <div className="mt-5 text-center mobileLandscape:text-start">
+          <Localized id="cs-recent-activity">
+            <Link
+              data-testid="settings-recent-activity"
+              className="link-blue text-sm"
+              to="/settings/security_events"
+            >
+              Recent Activity
```

Comment:
I thought about calling this section `Security Events` since that is how it is reference in code, but really it is just a list of recent account events and that sounds more user friendly.

Reply:
All the updates except database ones are an easy fix 👍🏽

Reply:
Sorry yeah I didn't mean update the DB table name, but was saying I wasn't sure if it was worth updating the _component_ names to match the new "recent account activity" over calling them "security events" as you are here. Seems good to me to name the page component "account activity" and leave the others just as you have 👍